Rakefile

```diff
@@ -1,8 +1,14 @@
-require 'bundler/gem_tasks'
+require "bundler/gem_tasks"
+
+desc "Appraisals: Test different dependency versions"
+task :run_appraisals do
+  sh("bundle exec appraisal install")
+  sh("bundle exec appraisal rspec --tag ~type:black_box")
```

Comment:
The appraisals run all the specs except the JetBlack black box tests, because the `BUNDLE_GEMFILE` env variable set by Appraisal to switch Gemfiles would be ignored anyway. This is because the black box tests run in their own clean bundler environment.